metrics/web/lib/dashboard/presentation/widgets/metrics_table_row.dart

```diff
@@ -59,19 +59,19 @@ class MetricsTableRow extends StatelessWidget {
           status,
           Flexible(
             child: SizedBox(
-              width: 326.0,
+              width: 334.0,
```

Comment:
Should we maybe rethink this moment as there is a `28.0` padding from the box with a name to the build results graph? Also, it appears that the whole tile has symmetric horizontal `32.0` padding, but now it is `const EdgeInsets.only(left: 32.0, right: 26.0)`.
![image](https://user-images.githubusercontent.com/70939496/108200930-ceac6580-7127-11eb-8158-e9922b7220c3.png)
![image](https://user-images.githubusercontent.com/70939496/108201040-f4396f00-7127-11eb-81d9-64e270920145.png)